docs/3.0.0-beta.x/guides/deployment.md

```diff
@@ -1562,3 +1562,11 @@ git commit -am "Changes to my-project noted"
 git push heroku master
 heroku open
 ```
+
+---
```

Comment:
Can you remove this `---`
And the previous one that is for `### Project updates`